src/Compilers/CSharp/Test/Semantic/Semantics/RefLocalsAndReturnsTests.cs

```diff
@@ -990,6 +990,19 @@ void M()
                 Diagnostic(ErrorCode.ERR_RefLocalOrParamExpected, "P").WithLocation(8, 9));
         }
 
+        [Fact, WorkItem(44153, "https://github.com/dotnet/roslyn/issues/44153")]
+        public void RefErrorProperty()
+        {
+            CreateCompilation(@"
+public class C {
+    public ref ERROR Prop => throw null!;
+}
+").VerifyEmitDiagnostics(
```

Comment:
>VerifyEmitDiagnostics [](start = 3, length = 21)

Does this hit the failure? From the issue description, it's not clear.

Reply:
I made a mistake in the issue description. It's now fixed. Thanks

Reply:
Confirmed this fails without the change to DefiniteAssignment.

---
In reply to: [424042228](https://github.com/dotnet/roslyn/pull/44186#discussion_r424042228) [](ancestors = 424042228)